src/components/OmsInstanceNavigation.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
We can the name the component OmsInstanceNavigator instead (like ProductIdentitifer or LanguageSwitcher).